django/publicmapping/publicmapping/settings.py

```diff
@@ -75,6 +79,29 @@
     'django.middleware.locale.LocaleMiddleware',
 ]
 
+ROLLBAR_CLIENT_TOKEN = os.getenv('ROLLBAR_CLIENT_TOKEN', None)
+ROLLBAR_SERVER_TOKEN = os.getenv('ROLLBAR_SERVER_TOKEN', None)
+ROLLBAR = None
+
+if ENVIRONMENT in ['Staging', 'Production']:
+    xray_recorder.configure(plugins=["EC2Plugin"])
+    patch_all()
+    MIDDLEWARE = ['aws_xray_sdk.ext.django.middleware.XRayMiddleware'] + MIDDLEWARE
+    INSTALLED_APPS += ['aws_xray_sdk.ext.django']
+
+    # Rollbar setup
+    ROLLBAR = {
+        'access_token': ROLLBAR_SERVER_TOKEN,
+        'environment': ENVIRONMENT,
+        'branch': 'master',
+        'root': '/usr/src/app',
+    }
+    MIDDLEWARE.append('rollbar.contrib.django.middleware.RollbarNotifierMiddleware')
+
+SERIALIZATION_MODULES = {
```

Comment:
Based on the blame in the other repository, this change was added as part of a management command that doesn't exist here. Based on that, it doesn't seem needed. May need to get feedback from @pcaisse to confirm.

See: https://github.com/azavea/district-builder-dtl-pa/commit/ad11eadd6059adb1f5b20f805c1433aac0dc846e

Reply:
After looking at the history, and inspecting the contents of https://github.com/PublicMapping/DistrictBuilder/pull/553, I don't think there's anything that would depend on this.